openquake/qa_tests_data/gmf_ebrisk/case_4/job_haz.ini

```diff
@@ -0,0 +1,46 @@
+[general]
+description = event based two source models
+calculation_mode = event_based
+structural_vulnerability_file = structural_vulnerability_model.xml
+
+[exposure]
+exposure_file = exposure_model.xml
+
+[site_params]
+reference_vs30_type = measured
+reference_vs30_value = 760.0
+reference_depth_to_2pt5km_per_sec = 5.0
+reference_depth_to_1pt0km_per_sec = 100.0
+
+[erf]
+width_of_mfd_bin = 0.1
+rupture_mesh_spacing = 2.0
+area_source_discretization = 10
+
+[logic_trees]
+source_model_logic_tree_file = source_model_logic_tree.xml
+gsim_logic_tree_file = gsim_logic_tree.xml
+
+[correlation]
+ground_motion_correlation_model = JB2009
+ground_motion_correlation_params = {"vs30_clustering": True}
+
+[hazard_calculation]
+#intensity_measure_types = PGA, SA(0.1), SA(0.3), SA(0.4)
```

Comment:
Should this be removed?

Reply:
Oh, yes, this is a leftover, I missed that!